### PR TITLE
Build out ancillary pages and nav

### PR DESCRIPTION
**Pull request by saragibby**
Adds:
- top nav
- contact us page
- plan your trip page
- blog listing page

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/blog**](https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/blog) | [**/wi/become-a-partner**](https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/become-a-partner) |
|:-:|:-:|
|<a href="https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/blog">Screenshot of /blog</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/INYzm9EeG75iND8vw2aCP61GkPM_">(view full size)</a> |<a href="https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/become-a-partner">Screenshot of /wi/become-a-partner</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/K7s61zZxVjizD4CiWvNckmmCjPU_">(view full size)</a> |



| [**/wi/contact**](https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/contact) | [**/wi/partners**](https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/partners) |
|:-:|:-:|
|<a href="https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/contact">Screenshot of /wi/contact</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/QNpQD_FBtK2Uqyvbb_nsT45TTcE_">(view full size)</a> |<a href="https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/partners">Screenshot of /wi/partners</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/A1iE56R2TDmE_RwswkIX_w10Jgs_">(view full size)</a> |



| [**/wi/plan-your-trip**](https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/plan-your-trip) |
|:-:|
|<a href="https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh/wi/plan-your-trip">Screenshot of /wi/plan-your-trip</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/P5of8Q9pUS4jGYuiJZWHoK7xsls_">(view full size)</a> |







Commit 7839937bebee3ebce3e75db0fbbee26b2973c909 (https://that-website-git-sara-build-out-pages-and-nav.that-conference.now.sh).

File: components/BlogList/BlogListItem.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import fm from 'front-matter';
+import Error from '../../pages/_error';
+import ImageContainer from '../shared/ImageContainer';
+import { below } from '../../utilities/breakpoint';
+
+const BlogContainer = styled(ImageContainer)`
+  width: 70vw;
```
Comment (kburnell) [inline components/BlogList/BlogListItem.js]: OMG!  Just looked up what `vw` actually did (I know noob), but why aren't we using this more?

File: components/Header/Nav.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
+import styled from 'styled-components';
+import React, { useState } from 'react';
+
+import NavItem from './NavItem';
+// import Icon from '../shared/Icon';
```
Comment (kburnell) [inline components/Header/Nav.js]: looks like this line can be removed

File: pages/blog.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <Head>
+      <title key="title">Blog - THAT Conference</title>
+    </Head>
+
+    <ContentSection>
+      <Grid columns={gridRepeat.xsmall}>
+        <Cell>
+          <h1 style={{ marginBottom: '0.5rem' }}>THAT Blog</h1>
+          <p className="medium-body-copy">
+            A collection of articles, tutoirals, stories all written by THAT
```
Comment (kburnell) [inline pages/blog.js]: *tutorials*

File: pages/blog.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            Crew and amazing member of THAT Community.
```
Comment (kburnell) [inline pages/blog.js]: *members*

File: pages/wi/contact.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+            hear it!
+          </p>
+          <LinkButton
+            href="mailto:hello@thatconference.com"
+            label="hello@thatconference.com"
+          />
+        </StyledImageContainer>
+        <StyledImageContainer>
+          <Title>Partnerships</Title>
+          <p>
```
Comment (kburnell) [inline pages/wi/contact.js]: this sentence read weird.  "...or if interested in becoming a partner." maybe?

File: pages/wi/plan-your-trip.js [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+      <ContentSection>
+        <Grid columns={gridRepeat.xsmall} alignContent="center">
+          <StyledImageContainer>
+            <Title>When</Title>
+            <p style={{ flexGrow: '2' }}>
+              {formattedStartDate} -
+              <br />
+              {formattedEndDate}
+            </p>
+            <LinkButton href={DEFAULT_WIP_PAGE} label="Ticket Options" />
```
Comment (kburnell) [inline pages/wi/plan-your-trip.js]: DEFAULT_WIP_PAGE is 404'ing because it is going to `/wi/work-in-progress`

Comment (saragibby) [reply pages/wi/plan-your-trip.js]: Thanks for catching! Fixed until I can get the ticker page added.

File: pages/wi/contact.js (hunk 5 above)
Comment (saragibby) [reply pages/wi/contact.js]: cooked up some new wordz here

**Comment by saragibby** [comment]
Thanks @kburnell! I love that your keyboard gave you immediate notification this was ready! 😄

**Pull request merged**